### ISSUE TITLE
New color scheme for the app

### ISSUE THREAD

**Issue by justdvnsh** [justdvnsh/AnimeClassroom#1]
This is an open discussion. We want a new and vibrant color scheme for the app. The colors should be complementary for both light and dark theme for the app

**Comment by vishalram14** [comment]
Hey, can I work on this?

**Comment by justdvnsh** [comment]
Hi, @vishalram14 . We are so happy to have you here. I have assigned this issue to you. Happy coding.
Also, please star the repo, so that this project could help much more people during the hacktoberfest and also, help in their open source journey. Thanks

**Comment by justdvnsh** [comment]
@vishalram14 Also, you can have multiple color schemes, and we can discuss it here, as to which one looks the best.

**Comment by vishalram14** [comment]
Will post the colors as soon as I am done with em

**Comment by vishalram14** [comment]
![HighContrast](https://user-images.githubusercontent.com/89023989/135839038-cf214c9b-84d5-45d6-903a-0f8c631a3c27.jpeg)

**Comment by vishalram14** [comment]
High contrast theme

**Comment by vishalram14** [comment]
Tell me if it is ok or anything that you specifically want

**Comment by justdvnsh** [comment]
@vishalram14 Nah ! This looks too bold. We would be going for some subtle colors. Let's keep it very minimal. Like some shade which would go with both white and black backgrounds. What do you think about it ?

**Comment by vishalram14** [comment]
Cool will work on it

**Comment by vishalram14** [comment]
![DarkTheme](https://user-images.githubusercontent.com/89023989/135843346-0fe136a1-aa9f-4fbe-bcfa-9f6077ca1857.jpeg)
![LightTheme](https://user-images.githubusercontent.com/89023989/135843353-4890843b-783f-473e-a888-c8d11713a601.jpeg)

**Comment by justdvnsh** [comment]
@vishalram14 . Maybe the second one. Not exactly this, but maybe some scheme related to this.

**Comment by vishalram14** [comment]
Ok

**Comment by vishalram14** [comment]
what changes would you like in the second one then?

**Comment by vishalram14** [comment]
![Light](https://user-images.githubusercontent.com/89023989/135844583-020e598d-dbf6-44eb-bd32-c85bb4bef52e.jpeg)

**Comment by vishalram14** [comment]
![dark](https://user-images.githubusercontent.com/89023989/135845527-7df6113d-0a13-467c-8f1c-ee3887948b77.jpeg)

**Comment by justdvnsh** [comment]
Yeah the teal color with black looks good. Let's make it a darker green. For both dark and light themes. @vishalram14

**Comment by vishalram14** [comment]
make which one darker for the light theme ?

**Comment by vishalram14** [comment]
the manga activity screen doesn't display anything, so am not sure what to do there

**Comment by justdvnsh** [comment]
make the green color darker. Like instead of teal, let's make that somewhat dark shade of green, for both, light and dark themes.

**Comment by justdvnsh** [comment]
> the manga activity screen doesn't display anything, so am not sure what to do there

It's fine. Just change the colors in the themes. The other screens would automatically take up the color.

**Comment by vishalram14** [comment]
so do you want me to replace the Blue in light theme also with dark green?

**Comment by justdvnsh** [comment]
> so do you want me to replace the Blue in light theme also with dark green?

Yup. I think it would look good.

**Comment by vishalram14** [comment]
Ok

**Comment by vishalram14** [comment]
![alternate](https://user-images.githubusercontent.com/89023989/135855949-eeaf469e-38a2-4055-bd20-ee263b08411f.jpeg)

**Comment by vishalram14** [comment]
What exactly is the recycler item here?

**Comment by vishalram14** [comment]
![dark alernate](https://user-images.githubusercontent.com/89023989/135858715-29d86340-173c-493b-bd0c-15ad8fd9c478.jpeg)

**Comment by vishalram14** [comment]
Anymore Changes , have posted both the light and dark themes here, will send a PR if confirmed

**Comment by vishalram14** [comment]
Waiting for confirmation

**Comment by justdvnsh** [comment]
let's discuss it on slack. Join the slack channel @vishalram14 . I need to have other peoples confirmation as well.

**Comment by vishalram14** [comment]
Could you just finalize the colors with your team and inform me

**Comment by justdvnsh** [comment]
@vishalram14 There's no team man. There's only you guys. The contributors are team 😂. I am just waiting for them to confirm if they like the color scheme.

**Comment by vishalram14** [comment]
Create a poll for the colours they want for the app background, the drawer and the circles(genre background)
Will then change the colours accordingly. This is the fastest way I can think of.

**Comment by vishalram14** [comment]
Any updates?

**Comment by vishalram14** [comment]
These are the final colour schemes (both light and dark)
![Screenshot_1633537543](https://user-images.githubusercontent.com/89023989/136245073-ca7d92ca-7d86-4cd3-aeee-91f986b6b5ec.png)

![Screenshot_1633537519](https://user-images.githubusercontent.com/89023989/136245079-454726ea-cddd-42b1-91ea-99c58ce675f3.png)
![Screenshot_1633537605](https://user-images.githubusercontent.com/89023989/136245236-0f50ac80-673a-4d67-b31b-861c6c0f30dc.png)
![Screenshot_1633537619](https://user-images.githubusercontent.com/89023989/136245249-b93beda3-7e1a-475e-9204-8d9c174e1dc3.png)

**Comment by vishalram14** [comment]
These are the final colour schemes ( for both light and dark themes

![Light_Home](https://user-images.githubusercontent.com/89023989/136245997-c899c648-c83a-4acc-ab4b-c392de115d80.png)
![Light_2ndActivity](https://user-images.githubusercontent.com/89023989/136246008-296e0a7b-2088-4277-87e8-285a526f4224.png)
![Dark_Home](https://user-images.githubusercontent.com/89023989/136246015-0a7a3314-7b92-4d20-80c2-a4111b309385.png)
)
![Dark_2ndActivity](https://user-images.githubusercontent.com/89023989/136245985-fe0c6a6c-74a0-4719-8fb8-e6dee544a3ff.png)

**Comment by Rishavgupta12345** [comment]
can i work on this ?
